x-pack/plugins/alerting/server/alerts_client/alerts_client.ts

```diff
@@ -402,6 +416,59 @@ export class AlertsClient<
     return this.legacyAlertsClient.factory();
   }
 
+  public async getPersistentAlerts({
+    ruleId,
+    spaceId,
+    excludedAlertInstanceIds,
+    alertsFilter,
+    start,
+    end,
+    executionUuid,
+    formatAlert,
```

Comment:
This is defined on the rule type under the `ruleType.alerts` which we have access to already in this client so it shouldn't need to be passed in.

Reply:
done.